mokapot/tabular_data.py

```diff
@@ -119,11 +121,13 @@ def _get_orig_columns(self, columns: list[str] | None) -> list[str] | None:
         return orig_columns
 
     def _get_mapped_dataframe(self, df: pd.DataFrame) -> pd.DataFrame:
-        # todo: enable this again...
+        # todo: enable this again... Q June2024: When?
```

Comment:
When should we enable this again?

Reply:
Instead of creating a copy of a dataframe (like in this function), it would be desirable to instead change an existing dataframe for efficiency. When trying this, this caused problems, that we haven't fully resolved yet. We'll do the change when this part is tested and fool-proof.